packages/wonder-blocks-dropdown/src/components/single-select.stories.js

```diff
@@ -167,6 +167,23 @@ export const WithFilterOpened: StoryComponentType = () => (
     <SingleSelectWithFilter opened={true} />
 );
 
```

Comment:
I added this story because I wanted to see if the search field automatically receives focus if the filter is open and no value is selected. It does.

Reply:
Not sure why the open dropdown isn't showing up in the snapshot. I guess it's not there on initial render? Should I disable the snapshot?

Reply:
oh that's an issue with Chromatic not being able to capture some "async" snapshots (we have seen this already happening with some Tooltip and Popover stories that we had to disable for now).

One thing that you can try out is to use the [delay](https://www.chromatic.com/docs/delay) feature and see if that fixes the issue.

Reply:
Hmm... the delay doesn't seem to be helping. I'm wondering if the current on chromatic the same as it is for the `WithFilterOpened` story

Reply:
I updated the story so that it uses the same class as the other ones, and now the chromatic issue is gone!